src/agent/taskmanager.ts

```diff
@@ -116,11 +116,12 @@ export class TaskManager {
     private getTaskDefinitionsUri(variables: { [key: string]: string }): string {
         var taskDefinitionsUri = variables["system.taskDefinitionsUri"];
         if (!taskDefinitionsUri) {
-            taskDefinitionsUri = variables[cm.sysVars.collectionUri];
-        }
-        if (!taskDefinitionsUri) {
+            // this version of the agent does not use vso-node-api, but it may run against servers that don't support the collection-level tasks
+            // system.taskDefinitionsUri was added in the same sprint as collection-level tasks...
+            // so if it's not there we can assume collection-level tasks aren't supported
             taskDefinitionsUri = this.context.config.settings.serverUrl; 
```

Comment:
This is specifically for QU1 - but it should work for RTM since RTM doesn't send either of those variables down.  I'll set up another VM and verify on Monday.